File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/Flag.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class Flag : DomainResource, IResource
{
    public required Base.CodeableConcept Code { get; set; }
    public Base.ResourceReference? Author { get; set; }
    public Base.Period? Period { get; set; }
    public required string Status { get; set; }
    public required Base.ResourceReference Subject { get; set; }
    public Base.CodeableConcept[]? Category { get; set; }
    public Base.ResourceReference? Encounter { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Flag.py
from typing import Optional, List
from base import Period
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

class Flag(DomainResource):
    code: CodeableConcept
    author: Optional[Reference] = None
    period: Optional[Period] = None
    status: str
    subject: Reference
    category: Optional[List[CodeableConcept]] = None
    encounter: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = None